### PR TITLE
Added deployTime to about

### PR DESCRIPTION
**Pull request by wheresrhys**
for diagnosing if your rebuild of a commit is being served or the previous deploy of that build

@matthew-andrews

### CODE REVIEW

**Comment by matthew-andrews** [comment]
yup good idea :+1:

**Comment by matthew-andrews** [comment]
strictly speaking though this isn't deploy time, it's build time.

**Comment by matthew-andrews** [comment]
( in the future we might split those two things )

File: lib/about.js [hunk 1]
```diff
@@ -17,7 +17,8 @@ module.exports = function(opts) {
 				description: name,
 				support: 'next.team@ft.com',
 				supportStatus: 'active',
-				appVersion: commit
+				appVersion: commit,
+				deployTime: new Date().toString()
```
Comment (commuterjoy) [inline lib/about.js]: `new Date().toISOString()` is more parseable by future systems that might use this data

**Comment by wheresrhys** [comment]
I was splitting that hair myself as soon as I'd committed. Will rename and ISO it up

**Comment by matthew-andrews** [comment]
:+1:

**Pull request merged**